### DISCUSSION TITLE
Contribution Guideline

### DISCUSSION THREAD

**Discussion by aarcex3** [illuin-tech/opyoid#132] [Q&A] [answered]
Hi, I like this project and I would like to contribute.
Is there any contribution guideline that you guys have?

Thanks

**Comment by Aegdesil** [comment, marked as answer]
Hi, thank you for your interest in this project !

Contributors are welcome to participate, I invite you to create discussions for feature requests that you would like to add to this library.
There are also some rules in the [CONTRIBUTING.md](https://github.com/illuin-tech/opyoid/blob/master/CONTRIBUTING.md) file for the more technical aspects.
